content/post/2025-05-20-update-of-math-fma-in-golang.markdown

```
@@ -0,0 +1,179 @@
+
+### 修正内容
+
+そんなに長くないのでパッチの内容を全部載せてしまいます。
+
+```diff
+diff --git a/src/math/all_test.go b/src/math/all_test.go
+index c253b7bc025d34..4e5f4517629dd8 100644
+--- a/src/math/all_test.go
++++ b/src/math/all_test.go
+@@ -2126,6 +2126,11 @@ var fmaC = []struct{ x, y, z, want float64 }{
+ 	// Issue #61130
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 2

(MD010, no-hard-tabs)